README.md

```diff
@@ -36,6 +36,7 @@ helm plugin install https://github.com/reactiveops/reckoner
       * `--helm-args <helm-arg>`: Pass arbitrary flags and parameters onto helm
         such as `--helm-args --set=foo=toast`, or `--helm-args --recreate-pods`.
         Multiples are supported but only one parameter per `--helm-args` is supported.
+        Note that specifying this flag will override `helm_args` in the course.yml file.
```

Comment:
Not for this PR but something to actively think about as we move this forward:

We may want to define a common precedence for arguments defined in each "level" and the order of override. This will help the code use common implementations for which argument to keep or override

Example - 1 = lowest precedence and all higher numbers override
1. course.yml
2. environment variables
3. cli arguments